components/PetList.jsx

```diff
@@ -21,7 +22,7 @@ export default function PetList({ onSelectPet, selectedPet }) {
                   onSelectPet(pet);
                 }}
               >
-                {pet.type}
+                <SVGIcon $variant={pet.type} size={100} ariaLabel={pet.type} />
```

Comment:
I am sorry I left the comment for the prop name prefixes with "$" which only applies to styled components. As you are not even using styled components on the SVGIcon.. it is safe to say, that I was completely wrong :D